vcgencmd/vcgencmd-completion.bash

```diff
@@ -0,0 +1,10 @@
+_vcgencmd() {
+    local cur prev words cword split
+    _init_completion -s || return
+    if ! ((cword == 1)); then
+        return
+    fi
+    local cmds=$(vcgencmd commands | sed 's/^.*=//;s/[",]//g')
+    COMPREPLY+=($(compgen -W "$cmds" -- $cur))
+} &&
```

Comment:
What does `&&` do in this context? The `pinctrl` completion has nothing here.

Reply:
Probably nothing. This is something I was sent quite a while ago. I only modified it a little to make it work a bit better, but otherwise I didn't question is too much.

If I take it out, it still seems to work, so... I'll do that.